### PR TITLE
4.0rc1 release info

### PR DESCRIPTION
**Pull request by raffenet**
## Pull Request Description

Updates to CHANGES and version.m4.

## Author Checklist
* [x] **Provide Description** 
      Particularly focus on _why_, not _what_. Reference background, issues, test failures, xfail entries, etc.
* [x] **Commits Follow Good Practice**
      Commits are self-contained and do not do two things at once. 
      Commit message is of the form: `module: short description` 
      Commit message explains what's in the commit.
* [ ] **Passes All Tests**
      Whitespace checker. Warnings test. Additional tests via comments.
* [x] **Contribution Agreement**
      For non-Argonne authors, check [contribution agreement](http://www.mpich.org/documentation/contributor-docs/). 
      If necessary, request an explicit comment from your companies PR approval manager.

### CODE REVIEW

**Comment by raffenet** [comment]
Updated to add a commit to fix an F08 binding issue. I think this is going to be it for 4.0rc1. I'd rather do a small rc2 after the break than try to get last minute fixes in before Thursday.
test:mpich/ch4/most
test:mpich/ch3/most

File: CHANGES [hunk 1]
```diff
@@ -18,7 +18,9 @@
 # Generate C API interface functions including man page notes and error
   checking using Python scripts.
 
-# Generate Fortran mpi_f08 binding using Python scripts.
+# Generate Fortran bindings using Python scripts.
```
Comment (hzhou) [inline CHANGES:21]: f90 bindings is still not converted

**Review by hzhou — APPROVED** [review]
Looks good. Just 1 minor comment.

**Comment by raffenet** [comment]
test:mpich/ch3/most
test:mpich/ch4/most

File: CHANGES (hunk 1 above)
Comment (raffenet) [reply CHANGES]: OK, I listed them individually to be clearer.

**Review by yfguo — APPROVED** [review]

**Pull request merged**